sdk/elixir/dagger_codegen/test/dagger/codegen/elixir_generator/object_renderer_test.exs

```diff
@@ -228,4 +228,33 @@ defmodule Dagger.Codegen.ElixirGenerator.ObjectRendererTest do
       """ <- render_type(ObjectRenderer, "test/fixtures/objects/return-scalar.json")
     )
   end
+
+  test "return list of enums" do
```

Comment:
Looks like a test fixture is missing: https://github.com/dagger/dagger/actions/runs/10705239020/job/29680354047?pr=8332#step:4:1585

```
  1) test return list of enums (Dagger.Codegen.ElixirGenerator.ObjectRendererTest)
     test/dagger/codegen/elixir_generator/object_renderer_test.exs:232
     ** (File.Error) could not read file "test/fixtures/objects/return-list-of-enums.json": no such file or directory
     code: """ <- render_type(ObjectRenderer, "test/fixtures/objects/return-list-of-enums.json")
     stacktrace:
       (elixir 1.17.2) lib/file.ex:385: File.read!/1
       (dagger_codegen 0.1.0) test/support/renderer_case.ex:13: Dagger.Codegen.RendererCase.decode_type_from_file/1
       (dagger_codegen 0.1.0) test/support/renderer_case.ex:27: Dagger.Codegen.RendererCase.render_type/2
       test/dagger/codegen/elixir_generator/object_renderer_test.exs:257: (test)
```

Reply:
Ohhh, i forgot to commit that file. 🥲

But I’m not on the computer right now. Will looking into that in around 5 hours.

Reply:
Fixed and rebase. :)